server/db/models/user.js

```diff
@@ -20,6 +20,15 @@ var schema = new mongoose.Schema({
     coordinates: {
         type: [Number]
     },
+    normalizedLocation: {
+        type: String
+    },
```

Comment:
How are we storing location on user? Is it just the location they're at when they sign up? What if they move around or change location?